docs/client.md

```diff
@@ -519,8 +506,8 @@ const cli = require("@eik/cli");
 const result = await new cli.Meta(options).run();
```

Comment:
The form of all of these commands are outdated as well.
They look like this now:
```js
const result = await cli.meta(options);
```